org.eclipse.wildwebdeveloper.xml/src/org/eclipse/wildwebdeveloper/xml/internal/ui/messages.properties

```
@@ -23,6 +23,16 @@ XMLCatalogPreferencePage_OpenInEditorApplyAndEdit=Apply and Edit
 # XML CodeLens preference page
 XMLCodelensPreferencePage_codelens_enabled=Enable XML &CodeLens
 
+# XML Formatting preference page
+XMLFormattingPreferencePage_format_emptyElements=Expand/collapse empty elements
```

I suggest `Default visibility for empty elements`
```suggestion
XMLFormattingPreferencePage_format_emptyElements=Default visibility for empty elements
```

---

I would like to keep this label since we use it too in vscode-xml.